CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.2.2 (2024-01-05)
```

Comment:
Let's call this section `UNRELEASED` until we have an actual release tag.

Reply:
👍 I'll update it later once merged